lib/home/view/user_search_screen.dart

```diff
@@ -19,27 +18,30 @@ class UserSearchScreen extends StatelessWidget {
     }
     return LayoutBuilder(builder: (context, box) {
       return SafeArea(
-        child: Scaffold(
-          resizeToAvoidBottomInset: false,
-          appBar: CustomAppBar(
-            titleChildWidget: CircleAvatar(
-              backgroundImage: NetworkImage('${user.photo}'),
-            ),
-            actionChildWidget: [
-              RowHeaderIcons(),
-            ],
-          ),
-          body: SingleChildScrollView(
-            child: Column(
-              children: [
-                SearchInputWidget(
-                  box: box,
-                ),
-                UserSearchList(
-                  box: box,
-                ),
+        child: GestureDetector(
+          onTap: () {
+            // close keyboard
+            FocusScope.of(context).unfocus();
+          },
+          child: Scaffold(
+            resizeToAvoidBottomInset: false,
+            appBar: CustomAppBar(
+              titleChildWidget: CircleAvatar(
+                backgroundImage: NetworkImage('${user.photo}'),
+              ),
+              actionChildWidget: [
+                RowHeaderIcons(),
               ],
             ),
+            body: SingleChildScrollView(
```

Comment:
make the Recently sent and Contacts  section are not independently scrollable, you can achieve this with `CustomScrollView`